examples/Azure_IoT_Central_ESP32/Azure_IoT_Central_ESP32.ino

```diff
@@ -331,7 +339,17 @@ void setup()
   azure_iot_config.use_device_provisioning = true; // Required for Azure IoT Central.
   azure_iot_config.iot_hub_fqdn = AZ_SPAN_EMPTY;
   azure_iot_config.device_id = AZ_SPAN_EMPTY;
-  azure_iot_config.device_key = AZ_SPAN_FROM_STR(IOT_CONFIG_DEVICE_KEY);
+  
+  #ifdef IOT_CONFIG_USE_X509_CERT
```

Comment:
Wonderful. It's like if our minds were connected. It's also a good sign that it was intuitive to add that to the unimidd, perhaps a sign its design is proving itself good so far.